applications/FluidDynamicsApplication/python_scripts/operations/from_potential_flow_to_compressible_Navier_Stokes_operation.py

```diff
@@ -0,0 +1,72 @@
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as KratosCompressible
+
+class FromPotentialFlowToCompressibleNavierStokesOperation(KratosMultiphysics.Operation):
+
+    def __init__(self, model, settings):
+        super().__init__()
+
+        # Cannot validate as settings may differ among input types
+        settings.AddMissingParameters(self.__GetDefaultSettings())
+
+        # Declare required member variables
+        self.model = model
+        self.settings = settings
+
+    def Execute(self):
+
+        origin_model_part_name = self.settings["origin_model_part"].GetString()
+        destination_model_part_name = self.settings["destination_model_part"].GetString()
+        variable_names = self.settings["variable_names_to_pass"].GetStringArray()
+
+        # saving the modelparts
+        origin_model_part = self.model.GetModelPart(origin_model_part_name)
+        destination_model_part = self.model.GetModelPart(destination_model_part_name)
+
+        # computing nodal values
+        nodal_value_process = KratosCompressible.ComputeNodalValueProcess(origin_model_part, variable_names)
+        nodal_value_process.Execute()
+
+        # Transfer the potential flow values as initial condition for the compressible problem
+        tem_0 = 273
+        gamma = 1.4
+        c_v = 722.14
+        c = (gamma*(gamma-1.0)*c_v*tem_0)**0.5
+        free_stream_rho = 1.0
+        free_stream_mach = 0.8
```

Comment:
Same comment. Besides, I think that if these are not provided we can try to extract them from the origin potential model part. If they are not found, we throw an error.

Reply:
In the origin model part are all the parameters except the temperature. 

For the moment, this temperature could be in the parameters of the operation as reference temperature.